content/en/blogs/kubesphere-3.1.0-ga-announcement.md

```diff
@@ -0,0 +1,156 @@
+---
+title: 'Extending Kubernetes to Edge: KubeSphere Container Platform 3.1 General Availability'
+keywords: KubeSphere, GA
+description: The KubeSphere community announced the general availability of KubeSphere v3.1.0 which features edge node management, metering and billing, enhanced DevOps and more.
+tag: 'Kubernetes, KubeSphere, GA, Announcement'
+createTime: '2021-05-14'
+author: 'Sherlock'
+snapshot: '/images/blogs/en/kubesphere-3.1.0-ga-announcement/banner.png'
+---
+
+On April 29, 2021, the KubeSphere community was excited to announce the general availability of KubeSphere v3.1.0. KubeSphere, as an app-centric distributed operating system running on top of Kubernetes, has further expanded its portfolio to deliver more robust experiences for users across the globe. This has once again echoed our commitment: enabling our users to run Kubernetes workloads where and when they want with ease and security.
+
+KubeSphere v3.1.0 looks to provide an enabling environment for users **as they deploy production workloads not just across clouds but also at the edge**. Besides, the new **Metering and Billing** function helps you better understand infrastructure operating costs. Existing features of **multi-cluster management**, **multi-tenant management**, **observability**, **DevOps**, **application lifecycle management**, and **microservices governance** have also been enhanced as we work to ensure better user experiences **across clusters and clouds**.
+
+"KubeSphere has emerged as one of the most popular Kubernetes distributions," said Ray Zhou, Head of the KubeSphere project. "The release of KubeSphere 3.1 marks an important milestone as we further empower enterprises to gain first-mover advantages especially in edge computing."
```

Comment:
Is it better to consistently use `v3.1.0` instead of `3.1` throughout the whole article?

Reply:
I did not use `v3.1.0` here because in spoken language you can hardly hear anybody say `v3.1.0`. Instead, people will just say `KubeSphere 3.1`.